config/controllers/usres_controllers.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+end

```

Comment:
- [ ] `users_controller.rb` file suppose to be inside `./app/controllers` directory, not inside `./config/controllers/` as you did, and name it `users_controller.rb`, not `users_controllers.rb`. please fix that.